src/mapping/config/MappingConfiguration.cpp

```diff
@@ -247,6 +256,7 @@ MappingConfiguration::MappingConfiguration(
   // Add the relevant attributes to the relevant tags
   addAttributes(projectionTags, {attrFromMesh, attrToMesh, attrDirection, attrConstraint});
   addAttributes(rbfDirectTags, {attrFromMesh, attrToMesh, attrDirection, attrConstraint, attrPolynomial, attrXDead, attrYDead, attrZDead});
+  addAttributes(rbfGreedyTags, {attrFromMesh, attrToMesh, attrDirection, attrConstraint, attrPolynomial, attrXDead, attrYDead, attrZDead}); // TODO: Hier Tag für Greedy Typen registrieren. //TODO: prüfe ob attrPolynomial, ... notwendig?
```

Comment:
See above, we could definitely try to combine greedy and polynomials

Reply:
Added the parameter back.